src/super_gradients/training/utils/predict/predictions.py

```diff
@@ -20,14 +20,37 @@ class DetectionPrediction(Prediction):
     bboxes_xyxy: np.ndarray
     confidence: np.ndarray
     labels: np.ndarray
+    target_bboxes_xyxy: np.ndarray
+    target_labels: np.ndarray
 
-    def __init__(self, bboxes: np.ndarray, bbox_format: str, confidence: np.ndarray, labels: np.ndarray, image_shape: Tuple[int, int]):
+    def __init__(
+        self,
+        bboxes: np.ndarray,
+        bbox_format: str,
+        confidence: np.ndarray,
+        labels: np.ndarray,
+        image_shape: Tuple[int, int],
+        target_bboxes: np.ndarray,
+        target_labels: np.ndarray,
+        target_bbox_format: str,
+    ):
         """
         :param bboxes:      BBoxes in the format specified by bbox_format
         :param bbox_format: BBoxes format that can be a string ("xyxy", "cxywh", ...)
         :param confidence:  Confidence scores for each bounding box
         :param labels:      Labels for each bounding box.
         :param image_shape: Shape of the image the prediction is made on, (H, W). This is used to convert bboxes to xyxy format
+
+        :param target_bboxes: Optional[List[np.ndarray]], ground truth bounding boxes. Can either be an np.ndarray of shape
```

Comment:
all the types in the documentation and all shapes are wrong

Reply:
0798978413fab364fd7d89ea23baeab266c80e77 fixed